tests/memory_checker/test_memory_checker.py

```diff
@@ -268,8 +264,8 @@ def test_setup_and_cleanup(duthosts, creds, enum_dut_feature_container,
 
 
 @pytest.fixture
-def remove_and_restart_container(duthosts, creds, enum_dut_feature_container,
-                                 enum_rand_one_per_hwsku_frontend_hostname):
+def remove_and_restart_container(duthosts, creds, enum_rand_one_per_hwsku_frontend_hostname,
+                                 enum_rand_one_asic_index, enum_dut_feature):
     """Removes and restarts 'telemetry' container from DuT.
 
     Args:
```

Comment:
Minor comment: could change Args in the comment to reflect new list

Reply:
done, thanks!